spec/linters/coala/coala_json.json

```diff
@@ -0,0 +1,678 @@
+{
+  "results": {
+    "default": [
+      {
+        "additional_info": "",
+        "affected_code": [
+          {
+            "end": {
+              "column": null,
+              "file": "/home/mauricio/test-coala/ReadImage.java",
```

Comment:
Is this coupled to your machine? It seems to me that the file is not actually read below, but I just want to make sure I'm right.